modules/LaneDetection/src/LaneManager.cpp

```diff
@@ -40,15 +40,15 @@ namespace XyCar
             // std::cout << "stop: " << lane_state.stop_flag_ << std::endl;
 
             int32_t centor = static_cast<int32_t>((lane_state.right_pos_ + lane_state.left_pos_) / 2);
-            int32_t error = (k_frame_width / 2) - centor;
+            int32_t error = centor - (k_frame_width >> 1);
```

Comment:
```suggestion
            constexpr int32_t k_frame_width_half = k_frame_width >> 1;
            int32_t error = centor - k_frame_width_half;
```
조금 더 최적화를 한다면, `k_frame_width`가 정해졌으니, half도 constexpr로 지정할 수 있을겁니다